oshi-core/src/main/java/oshi/hardware/CentralProcessor.java

```diff
@@ -339,6 +339,11 @@ public int getIndex() {
         }
     }
 
+
+    ProcessorIdentifier getProcessorIdentifier();
```

Comment:
This is good here but you haven't implemented it in the AbstractCentralProcessor class, so you're failing compilation.  Also you need a Javadoc for this, something like 

> Returns the CPU's identifier strings such as the CPU's stepping, model, and family information (also called the signature of a CPU).